src/components/SelectionList/Search/TransactionListItemRow.tsx

```diff
@@ -1,5 +1,5 @@
 import {Str} from 'expensify-common';
```

Comment:
We also need to pass the prop to `ActionCell` in [ReportListItemRow](https://github.com/Expensify/App/blob/main/src/components/SelectionList/Search/ReportListItem.tsx#L203-L208)